ui/values/localizable/en.go

```diff
@@ -22,7 +22,7 @@ const EN = `
 "currentTotalBalance" = "Current Total Balance";
 "walletStatus" = "Wallet Status";
 "blockHeaderFetched" = "Block header fetched";
-"noTransactionsYet" = "No transactions yet.";
+"noTransactionsYet" = "No transactions";
```

Comment:
```suggestion
"noTransactions" = "No transactions";
```
This will also need to be updated for every instance where it is used.

Reply:
Ok, updated